src/harness/sas_objects.py

```diff
@@ -0,0 +1,353 @@
+#    Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+import json
+import os
+import sas
+import urllib
+
+
+class Grant:
+
+    """
+    A Grant object has the following methods
```

Comment:
This is likely to fall out of date if/when we add new methods. Either remove this docstring or replace with a high-level description of how the class should be used and what it's intending to accomplish.

Reply:
Added   high level description for all the classes.